libs/auth/env.go

```diff
@@ -38,3 +38,22 @@ func GetEnvFor(name string) (string, bool) {
 
 	return "", false
 }
+
+// EnvVars returns the list of environment variables that the SDK reads to configure
+// authentication.
+// This is useful for spawning subprocesses since you can unset all auth environment
+// variables to clean up the environment before configuring authentication for the
+// child process.
+func EnvVars() []string {
+	out := []string{}
+
+	for _, attr := range config.ConfigAttributes {
+		if len(attr.EnvVars) == 0 {
+			continue
+		}
+
+		out = append(out, attr.EnvVars[0])
```

Comment:
Why do we take only first element here?

Reply:
Thanks! It does not matter right now since there's at max one env var per auth configurable. But we should indeed gather all of them.